prow/plugins/override/override_test.go

```diff
@@ -345,6 +362,25 @@ func TestHandle(t *testing.T) {
 			},
 			checkComments: []string{"on behalf of " + adminUser},
 		},
+		{
+			name:    "successfully override unknown context derived from checkruns",
+			comment: "/override failure-checkrun",
+			checkruns: &github.CheckRunList{
+				CheckRuns: []github.CheckRun{
+					{Name: "incomplete-checkrun"},
+					{Name: "failure-checkrun", CompletedAt: "1800 BC", Conclusion: "failure"},
+					{Name: "failure-checkrun", CompletedAt: "1800 BC", Conclusion: "success"},
```

Comment:
I don't understand what this tests. I think we end up creating an override since we always do that for failed checkruns but it is not visible in the testcase, because there is already a successful one?

Reply:
I haven't been able to write a good test case.

Basically, before execution, we have 2 checkruns called incomplete-checkrun, and failure-checkrun. After execution, we have an extra checkrun called failure-checkrun with Conclusion=success

Reply:
No, `checkruns` and `expectedCheckRuns` is the same. There is already a successful one for `failure-checkrun` in the beginning, maybe that is an accident?